src/main/java/edu/cornell/kfs/module/cam/rest/resource/CuCapAssetInventoryApiResource.java

```diff
@@ -184,18 +187,44 @@ public Response updateAsset(@PathParam(CuCamsConstants.CapAssetApi.ASSET_TAG_PAR
             return ex instanceof BadRequestException ? respondBadRequest() : respondInternalServerError(ex);
         }
     }
+    
+    @GET
+    @Path("/campusCodes")
+    public Response getCampusCodes(@Context HttpHeaders headers) {
+        try {
+            List<Map<String, String>> campusCodes = new ArrayList<>();
+            
+            Map<String, Object> fieldValues = new HashMap<>();
+            fieldValues.put(CuCamsConstants.CapAssetApi.ACTIVE, true);
+            Collection<org.kuali.kfs.sys.businessobject.Campus> campuses = 
+                SpringContext.getBean(BusinessObjectService.class).findMatching(
+                    org.kuali.kfs.sys.businessobject.Campus.class, fieldValues);
+            
+            for (org.kuali.kfs.sys.businessobject.Campus campus : campuses) {
+                Map<String, String> campusMap = new HashMap<>();
+                campusMap.put("value", campus.getCode());
+                campusMap.put("label", campus.getCode());
```

Comment:
You may want to check with the functionals on whether the label should include both the campus code and the campus name, instead of containing merely the campus code.

Reply:
I have an open question to the functionals on this one